includes/functions.php

```diff
@@ -1047,6 +1047,25 @@ function tsml_get_meeting($meeting_id = false)
 	return $meeting;
 }
 
+//function: get feedback_url
+//called in tsml_get_meta
+function tsml_feedback_url($post)
+{
+    global $tsml_feedback_url;
+
+    if (isset($tsml_feedback_url)) {
+        $id = $post->ID;
+        $slug = $post->post_name;
+
+        $url = $tsml_feedback_url;
+
+        $url = str_replace('{{id}}', $id, $url);
+        $url = str_replace('{{slug}}', $slug, $url);
+    }
+    return esc_url_raw($url, ['http', 'https', 'mailto', 'tel', 'sms']);
```

Comment:
apologies for the delay on reviewing this! i'm getting the following message when editing a meeting:

```
 Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065

Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065

Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065

Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065

Warning: Undefined variable $url in /Users/jreisner/Sites/valet/wordpress/wp-content/plugins/12-step-meeting-list/includes/functions.php on line 1065
```

if `tsml_feedback_url` is not set, what should it return? `null`?